src/pages/iou/request/step/IOURequestStepTaxRatePage.js

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import TaxPicker from '@components/TaxPicker';
+import taxPropTypes from '@components/taxPropTypes';
+import transactionPropTypes from '@components/transactionPropTypes';
+import useLocalize from '@hooks/useLocalize';
+import compose from '@libs/compose';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import * as IOU from '@userActions/IOU';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import withFullTransactionOrNotFound from './withFullTransactionOrNotFound';
+import withWritableReportOrNotFound from './withWritableReportOrNotFound';
+
+const propTypes = {
+    /** Route from navigation */
+    route: PropTypes.shape({
+        /** Params from the route */
+        params: PropTypes.shape({
+            /** The type of IOU report, i.e. bill, request, send */
+            iouType: PropTypes.string,
+
+            /** The report ID of the IOU */
+            reportID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    /* Onyx Props */
+    /** Collection of tax rates attached to a policy */
+    policyTaxRates: taxPropTypes,
+
+    /** The transaction object being modified in Onyx */
+    transaction: transactionPropTypes,
+};
+
+const defaultProps = {
+    policyTaxRates: {},
+    transaction: {},
+};
+
+// this is the formulae to calculate tax
+const calculateAmount = (taxRates, selectedTaxRate, amount) => {
+    const percentage = _.find(OptionsListUtils.transformedTaxRates(taxRates), (taxRate) => taxRate.name === selectedTaxRate).value;
+    const divisor = percentage.slice(0, -1) / 100 + 1; // slice to remove % at the end; converts "10%" to "10"
+    return parseInt(Math.round(amount - amount / divisor), 10) / 100; // returns The expense amount of transaction
```

Comment:
```suggestion
    return parseInt(Math.round(amount - (amount / divisor)), 10) / 100;
```

Reply:
prettier will remove the parenthesis BTW, it takes into account division before subtraction.